rticonnextdds_connector/rticonnextdds_connector.py

```diff
@@ -254,6 +254,39 @@ def __init__(self):
 
 		self.max_integer_as_double = 2**53
 
+	def get_any_value(self, getter_function, connector, input_name, index, field_name):
+		number_value = ctypes.c_double()
+		bool_value = ctypes.c_int()
+		string_value = ctypes.c_char_p()
+		selection = ctypes.c_int()
+		retcode = getter_function(
+			connector,
+			ctypes.byref(number_value),
+			ctypes.byref(bool_value),
+			ctypes.byref(string_value),
+			ctypes.byref(selection),
+			tocstring(input_name),
+			index + 1,
+			tocstring(field_name))
+		_check_retcode(retcode)
+
+		if retcode == _ReturnCode.no_data:
+			return None
+
+		if selection.value == _AnyValueKind.connector_number:
+			return number_value.value
+		elif selection.value == _AnyValueKind.connector_boolean:
+			return bool_value.value
+		elif selection.value == _AnyValueKind.connector_string:
+			python_string = _move_native_string(string_value)
+			try:
```

Comment:
Add TODO here referencing CON-142 (This was added in my change but I think we should document it so it doesn't get forgotten)